src/simplewallet/simplewallet.cpp

```diff
@@ -4058,6 +4086,32 @@ void simple_wallet::on_skip_transaction(uint64_t height, const crypto::hash &txi
 {
 }
 //----------------------------------------------------------------------------------------------------
+boost::optional<epee::wipeable_string> simple_wallet::on_get_password(const char *reason)
+{
+  // can't ask for password from a background thread
+  if (!m_in_manual_refresh.load(std::memory_order_relaxed))
+  {
+    message_writer(console_color_red, false) << tr("Password needed - use the refresh command");
```

Comment:
This prevents the command `show_transfers` from recognizing unconfirmed incoming transfers:
```
[wallet 55RKrP]: show_transfers
Password needed - use the refresh command
Error: Failed to get pool state:Password is needed to compute key image for incoming monero
```
Like with `import_multisig_info`, it seems that `m_in_manual_refresh` should be set to `true` while in the command.